libmodernize/main.py

```diff
@@ -102,7 +106,21 @@ def main(args=None):
     logging.basicConfig(format='%(name)s: %(message)s', level=level)
 
     # Initialize the refactoring tool
-    unwanted_fixes = set(options.nofix)
+    unwanted_fixes = set()
+    splitfixes = []
+    for fix in options.nofix:
+        splitfixes.extend(fix.split(','))
+    for fix in splitfixes:
+        matched = None
+        for tgt in avail_fixes:
+            if tgt == fix or tgt.endswith(".fix_{}".format(fix)):
+                matched = tgt
+                unwanted_fixes.add(matched)
+        if matched is None:
```

Comment:
This could be done without using the `matched` variable, by using `break` and an `else` on the for loop (which is run if it never reached `break`). Personally I find this quite neat for this kind of case, but I know some people don't like the for/else construct, so it's up to you.

Reply:
This was a bit of insurance against multiple matches too, which is why I didn't put a `break` in to begin with (that I often forget about `else` when it comes to `for` loops in Python is, however, a sad testament to my `C/C++` upbringing. :smiley: )

Reply:
Fair enough.